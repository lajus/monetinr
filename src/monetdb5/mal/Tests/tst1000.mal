#Access BATs 

b:= bat.new(:oid,:int);
bat.append(b,2);
bat.setName(b,"mybat");
c:bat[:oid,:int]:= bbp.bind("mybat");
io.print(c);
