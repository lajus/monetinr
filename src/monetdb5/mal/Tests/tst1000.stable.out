stdout of test 'tst1000` in directory 'monetdb5/mal` itself:


# 07:58:52 >  
# 07:58:52 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/Linux/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB5/log" --set mapi_port=42158 --set sql_port=55708 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst1000.mal </dev/null
# 07:58:52 >  

# Monet Database Server V4.99.19
# Copyright (c) 1993-2004, CWI. All rights reserved.
# Compiled for i686-redhat-linux-gnu/32bit; dynamically linked.
# config:/ufs/mk/monet5/Linux/etc/MonetDB5.conf
# dbfarm:/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
##Access BATs 
#
#b:= new(:int,:int);
#insert(b,1,2);
#bat.setName(b,"mybat");
#c:= bbp.bind("mybat");
#print(c);
#x:= bbp.getNames();
#print(x);
#
function user.main():void;
#Access BATs  
    b := bat.new(:oid,:int);
    bat.append(b,2);
    bat.setName(b,"mybat");
    c:bat[:oid,:int]  := bbp.bind("mybat");
    io.print(c);
end main;
#-----------------#
# h	mybat	  # name
# int	int	  # type
#-----------------#
[ 0@0,	  2	  ]

# 07:58:52 >  
# 07:58:52 >  Done.
# 07:58:52 >  

